components/Common/ArtistBanned.js

```diff
@@ -1,24 +1,13 @@
-import { useEffect, useState } from 'react'
-import cachios from 'cachios'
+import { useEffect } from 'react'
 import { useIntl } from 'hooks/useIntl'
 import { flagColor, flagText } from 'constants/flag'
+import useProfileData from 'hooks/useProfileData'
 
 const ArtistBanned = ({ creatorId, className }) => {
-	const [artistData, setArtistData] = useState(null)
 	const { localeLn } = useIntl()
+	const artistData = useProfileData(creatorId)
 
-	useEffect(async () => {
-		if (creatorId) {
-			const profileRes = await cachios.get(`${process.env.V2_API_URL}/profiles`, {
-				params: {
-					accountId: creatorId,
-				},
-				ttl: 600,
-			})
-			const userProfile = profileRes.data.data.results[0]
-			setArtistData(userProfile)
-		}
-	}, [creatorId])
+	useEffect(() => {}, [creatorId])
```

Comment:
> fyi sebenernya untuk case yang useEffect asynchronous bisa di wrap ke named function aja sih
> 
> ```tsx
> useEffect(() => {
>   const fetchCollection = async () => {
>     const res = await axios(`${process.env.V2_API_URL}/collections`, {
>       params: {
>         collection_id: data.collection_id,
>       },
>     })
>     setColDetail(res.data.data.results[0])
>   }
>   fetchCollection()
> }, [])
> ```
> 
> instead of anonymous function. awal2 gua bingung soalnya ada `;` di awalan useEffectnya cuman itu gara2 prettier ternyata kan
> 
> ```tsx
> useEffect(() => {
>   ;(async () => {
>     const res = await axios(`${process.env.V2_API_URL}/collections`, {
>       params: {
>         collection_id: data.collection_id,
>       },
>     })
>     setColDetail(res.data.data.results[0])
>   })()
> }, [])
> ```

woke fixing hu